modules/dreamview/backend/teleop/teleop.cc

```diff
@@ -39,11 +41,36 @@ TeleopService::TeleopService(WebSocketHandler *websocket)
 void TeleopService::Start() {
   // TODO get topic names from proto
   // TODO update proto to get all modems' info combined with rank
-  modem_info_reader_ = node_->CreateReader<ModemInfo>(
-      "/apollo/networks/modems",
-      [this](const std::shared_ptr<ModemInfo> &msg) { UpdateModemInfo(msg); });
-  daemon_cmd_writer_ = node_->CreateWriter<DaemonServiceCmd>(
-      "/apollo/teleop/daemon_service/cmd");
+
+  modem0_info_reader_ = node_->CreateReader<ModemInfo>(
+      "/apollo/teleop/network/modem0",
+      [this](const std::shared_ptr<ModemInfo> &msg) { UpdateModem(0, msg); });
```

Comment:
better not use maggic number 0, 1, 2...
use macros or gflags

Reply:
I removed the number and put the modem id in a string. 

This is a change to the original code that used the provider to name the modem. 
I changed it because the provider (ex Verizon or AT&T) can be the same for several modems.

```
 teleop_status_["modems"][modem_info->provider()] = modem_info->technology();
```

see new code [here](https://github.com/ApolloAuto/apollo/pull/9963/commits/182cb16555cbc3398c3cd162c885f924f3d05207)